package.json

```diff
@@ -253,6 +259,13 @@
           "group": "9_configuration@20",
           "when": "view == javaProjectExplorer && viewItem =~ /java:project(?=.*?\\b\\+unmanagedFolder\\b)(?=.*?\\b\\+uri\\b)/"
         }
+      ],
+      "editor/title": [
+        {
+          "command": "java.formatterSettings.showTextEditor",
+          "when": "activeCustomEditorId == java.formatterSettingsEditor && !readOnlyMode",
```

Comment:
I think it's ok to show the readonly text editor when the setting points to a url

Reply:
+1 for VSCode will show readonly tips
![remote](https://user-images.githubusercontent.com/45906942/119307945-fcf98e00-bc9e-11eb-8134-16151cea0ab6.jpg)